tsconfig.base.json

```diff
@@ -329,6 +329,8 @@
       "@island.is/nest/validators": ["libs/nest/validators/src/index.ts"],
       "@island.is/nova-sms": ["libs/nova-sms/src/index.ts"],
       "@island.is/plausible": ["libs/plausible/src/index.ts"],
+      "@island.is/regulations": ["libs/regulations/src/index.ts"],
+      "@island.is/regulations/*": ["libs/regulations/src/dist/*"],
```

Comment:
I don't think you want any reference to dist in here

Reply:
yeah that "dist" name sounds kind of ominous. It's just the folder where I place the other "entry-point" files.
Renamed to "sub" now.

Reply:
Ah yeah I didn't pay attention here, I saw dist and my brain trigger, but good to rename it 👍